### PR TITLE
Greenkeeper/parse server s3 adapter 1.2.0

### PR DESCRIPTION
**Pull request by acinader**

### CODE REVIEW

**Comment by acinader** [comment]
@flovilmart this what you have in mind?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 ## Parse Server Changelog
 
+### 2.5.4
```
Comment (flovilmart) [inline CHANGELOG.md]: Use master as untagged for now

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
+[Full Changelog](https://github.com/ParsePlatform/parse-server/compare/2.5.4...2.5.4)
+
+#### New DeprecationNotice:
```
Comment (flovilmart) [inline CHANGELOG.md]: Typo, or mark it ‘Breaking Changes’

**Comment by flovilmart** [comment]
Perhaps should be 2.6.0

**Comment by acinader** [comment]
I'm a little confused by your comment on master.  Is this what you mean?

**Comment by flovilmart** [comment]
@acinader that's OK, let's go with that!

**Pull request merged**